detailed_docs/RC/OnRCStatus.md

```diff
@@ -0,0 +1,148 @@
+## Use Case 1: Notification about RC modules allocation during application’s registration
+
+**Main Flow:**
+
+_Pre-conditions:_  
+
+a.  Mobile device is connected to SDL  
+
+b.	RC functionality is allowed on HMI  
+
+c.	All modules are free  
+
+d.	RC functionality is allowed by policies  
+
+_Steps:_
+
+1.	RC application starts registration (upon a OnButtonPress or SetInteriorVehicleData request)
```

Comment:
why mention `upon a OnButtonPress or SetInteriorVehicleData request`?

Reply:
@yang1070, please see the updates in https://github.com/smartdevicelink/sdl_requirements/pull/89/commits/2e9692b47878e6f1b766e7fbd380673c29af74db